.github/workflows/licenses.yaml

```diff
@@ -19,6 +19,16 @@ jobs:
       - name: "Generate dependency information"
         shell: bash
         run: make generate
+      - name: "Save dependency changes if committer is dependabot"
+        uses: datawire/go-mkopensource/actions/save-dependabot-changes@aosorio/test-branch
```

Comment:
test-branch ?

Reply:
@shepz Good catch. Forgot to drop those commits after my test was done.
PR is clean now.